src/ai/configuration.cpp

```diff
@@ -48,9 +48,9 @@ void configuration::init(const game_config_view &game_config)
 	era_ai_configurations_.clear();
 	mod_ai_configurations_.clear();
 
-	const config &ais = game_config.child("ais");
-	default_config_ = ais.child("default_config");
-	if (!default_config_) {
+	const config &ais = game_config.mandatory_child("ais");
+	default_config_ = ais.mandatory_child("default_config");
+	if ((false)) {
```

Comment:
This also seems like a place that should use `optional_child`. In this case I guess It'd look something like "if(optional_child) { default_config_ = … }`